src/components/InputText/InputText.tsx

```diff
@@ -4,9 +4,10 @@ import cs from "classnames";
 import { InputTextProps } from "./InputText.types";
 
 const InputText: FC<InputTextProps> = ({
-  domID = "input-text",
-  dataTestId = "test-input-text",
   className,
+  domID = className?.concat("-input-text"),
```

Comment:
I don't think this is an ideal solution, for example if one element has a className "a b c" and the other one has className "c b a" - they will get different IDs even if you expect them to be the same.
I think you should leave this domID as it was, and change it only in InputText instances in UploadModal and SidebarError components